public/app/features/explore/ExploreGraph.tsx

```diff
@@ -6,63 +7,58 @@ import {
   createFieldConfigRegistry,
   DataFrame,
   dateTime,
-  Field,
   FieldColorModeId,
   FieldConfigSource,
   getFrameDisplayName,
   GrafanaTheme2,
+  LoadingState,
+  SplitOpen,
   TimeZone,
 } from '@grafana/data';
+import { PanelRenderer } from '@grafana/runtime';
+import { GraphDrawStyle, LegendDisplayMode, TooltipDisplayMode } from '@grafana/schema';
 import {
   Icon,
   PanelContext,
   PanelContextProvider,
   SeriesVisibilityChangeMode,
-  TimeSeries,
-  TooltipPlugin,
   useStyles2,
   useTheme2,
-  ZoomPlugin,
 } from '@grafana/ui';
-import { LegendDisplayMode, TooltipDisplayMode, GraphDrawStyle } from '@grafana/schema';
+import appEvents from 'app/core/app_events';
 import { defaultGraphConfig, getGraphFieldConfig } from 'app/plugins/panel/timeseries/config';
-import { ContextMenuPlugin } from 'app/plugins/panel/timeseries/plugins/ContextMenuPlugin';
-import { ExemplarsPlugin } from 'app/plugins/panel/timeseries/plugins/ExemplarsPlugin';
-import { css, cx } from '@emotion/css';
+import { TimeSeriesOptions } from 'app/plugins/panel/timeseries/types';
+import { identity } from 'lodash';
 import React, { useEffect, useMemo, useRef, useState } from 'react';
-import { getFieldLinksForExplore } from './utils/links';
 import { usePrevious } from 'react-use';
-import appEvents from 'app/core/app_events';
 import { seriesVisibilityConfigFactory } from '../dashboard/dashgrid/SeriesVisibilityConfigFactory';
-import { identity } from 'lodash';
-import { SplitOpen } from 'app/types/explore';
 
 const MAX_NUMBER_OF_TIME_SERIES = 20;
 
 interface Props {
   data: DataFrame[];
   height: number;
   width: number;
-  annotations?: DataFrame[];
   absoluteRange: AbsoluteTimeRange;
   timeZone: TimeZone;
-  onUpdateTimeRange: (absoluteRange: AbsoluteTimeRange) => void;
+  loadingState: LoadingState;
+  annotations?: DataFrame[];
   onHiddenSeriesChanged?: (hiddenSeries: string[]) => void;
-  tooltipDisplayMode: TooltipDisplayMode;
+  tooltipDisplayMode?: TooltipDisplayMode;
```

Comment:
I don't think this property is needed as it's fixed (Multi) anyways from what I see.

Reply:
My bad- I see it differs between logs and series.